principles/fp-011-locus-of-authority.md

```diff
@@ -3,6 +3,7 @@ layout: principle
 id: fp-011-locus-of-authority
 title: Locus of Authority (principle 11)
 ---
+CONTENTS: [Recommendations/Requirements](#recommendations-and-requirements) | [Implementation](#implementation) | [Examples/Counter-Examples](#examples) | [Criteria for Review](#criteria-for-review) | [Feedback/Discussion](#feedback-and-discussion) 
```

Comment:
we should really switch to using bootstrap-toc for autogenerating these. future work i might be able to take care of